### PR TITLE
Ros2 tf update

### PR DESCRIPTION
**Pull request by Aposhian**
## Related Issues & PRs
#105 
#95

## Summary of Changes
- configure tf frames with parameters not a single prefix
- send static transforms only once with a static transform broadcaster

## Validation
In progress

### CODE REVIEW

**Comment by Aposhian** [comment]
Ok @Samahu I have updated this to include the latest merged PR, so now this only adds the static broadcaster

**Comment by Samahu** [comment]
If static broadcast is the appropriate (for ROS2) then we don't need to add the static publisher in the C++ code.. we could just utilize the static publishers that are created when RVIZ is launched:

* In XML:
  - https://github.com/ouster-lidar/ouster-ros/blob/513c826bd59f2679575320abe1191ee208be8cd8/ouster-ros/launch/rviz.launch.xml#L21
  - https://github.com/ouster-lidar/ouster-ros/blob/513c826bd59f2679575320abe1191ee208be8cd8/ouster-ros/launch/rviz.launch.xml#L24

* In Python
  - https://github.com/ouster-lidar/ouster-ros/blob/513c826bd59f2679575320abe1191ee208be8cd8/ouster-ros/launch/rviz.launch.py#L36
  - https://github.com/ouster-lidar/ouster-ros/blob/513c826bd59f2679575320abe1191ee208be8cd8/ouster-ros/launch/rviz.launch.py#L43

Basically all we need to do is enable these static publisher regardless if RVIZ is launched or not.

**Comment by Samahu** [comment]
I take that back we need the C++ code since we don't have the transform until metadata is received.

**Review by Samahu — APPROVED** [review]
Looks good to me.

**Comment by Samahu** [comment]
@Aposhian if you don't have any other changes I am ready to merge

**Comment by Aposhian** [comment]
I don't have any more changes, but I am still testing to validate that it works as expected.

**Comment by Samahu** [comment]
> I don't have any more changes, but I am still testing to validate that it works as expected.

I appreciate any help!

**Comment by Aposhian** [comment]
I've confirmed this works, although it gave me a little bit of unexpected results due to #33. This PR should be good to merge!

**Comment by Samahu** [comment]
@Aposhian I went ahead and merged your PR into a branch so I would be able to document your changes and also disable the static publishers that I had for the rivz node.

**Comment by Aposhian** [comment]
Sounds good!
